src/Symfony/Bridge/Twig/Command/LintCommand.php

```diff
@@ -88,7 +89,8 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $io = new SymfonyStyle($input, $output);
 
         if (null === $twig = $this->getTwigEnvironment()) {
-            $io->error('The Twig environment needs to be set.');
+            $errorIo = $output instanceof ConsoleOutputInterface ? new SymfonyStyle($input, $output->getErrorOutput()) : $io;
+            $errorIo->error('The Twig environment needs to be set.');
```

Comment:
The Twig Bridge DebugCommand update is still missing here, even in case the suggestion about throwing an exception is not considered. ^^